### ISSUE TITLE
Allow item numbers to be less than 1

### ISSUE THREAD

**Issue by VytskaLT** [Phoenix616/InventoryGui#31]
#### Is your feature request related to a problem? Please describe.

The StaticGuiElement doesn't allow the number of items to be less than 1.

#### Describe the solution you'd like

Allow the number of items to be less than 1.

This is what happens when you do this:
![image](https://user-images.githubusercontent.com/35194709/116823742-da3bf400-ab8e-11eb-8dfd-0d1e4d137012.png)
It's a pretty cool effect, so that's why I would like to use it.

**Comment by Phoenix616** [comment]
Pretty sure using non-standard numbers can lead to odd behaviour with items in inventories hence why I blocked them. That would require some research if all supported version can to this without issues.

**Comment by VytskaLT** [comment]
> Pretty sure using non-standard numbers can lead to odd behaviour with items in inventories hence why I blocked them. That would require some research if all supported versions can do this without issues.

You're right, it can cause some glitches. But in my opinion, it would still be good if there was an "unsafe" method to do this.

**Comment by RedstoneFuture** [comment]
A zero as a number can be usefull for come GUI  design situations, but if it's deprecated / unsafe by minecraft, the using of it is questionable. One more design-problem: you are already not able to use the "1" as numer. So if you would like visualize a number, you may display "0", "2" , "3" ... but not "1" with a number. Non-standard numbers are red in all ways, I think. This can be a problem in your GUI designs too.

**Comment by VytskaLT** [comment]
> A zero as a number can be usefull for come GUI design situations, but if it's deprecated / unsafe by minecraft, the using of it is questionable. One more design-problem: you are already not able to use the "1" as numer. So if you would like visualize a number, you may display "0", "2" , "3" ... but not "1". Non-standard numbers are red in all ways, I think. This can be a problem in your GUI designs too.
> 
> Which kinds of glitches we get with non-standard numbers and which kinds of non-standard numbers are possible?

-127 to 127 is possible I think. If the number is 0 or negative then it can disappear from your inventory when clicking on it.
